src/Camera.js

```diff
@@ -300,21 +302,21 @@ export default class Camera extends Component {
 
   startPreview() {
     if (Platform.OS === 'android') {
-      const props = convertNativeProps(this.props);
-      CameraManager.startPreview({
-        type: props.type,
-      });
+      UIManager.dispatchViewManagerCommand(
+        findNodeHandle(this),
+        UIManager.RCTCamera.Commands.startPreview,
+      );
     } else {
       CameraManager.startPreview();
     }
   }
 
   stopPreview() {
     if (Platform.OS === 'android') {
-      const props = convertNativeProps(this.props);
-      CameraManager.stopPreview({
-        type: props.type,
-      });
+      UIManager.dispatchViewManagerCommand(
+        findNodeHandle(this),
```

Comment:
can we save ref like this commit https://github.com/expo/expo-sdk/commit/c8f80cc563a5415c84c7f8dd139a85fa5bdc6c97?

Reply:
Thanks for the feedback, thanks for your reactivity, and your great contribution in general :)